### PR TITLE
VIT-600 add region to client

### PR DESCRIPTION
**Pull request by pazsaragi**

### CODE REVIEW

File: tests/api/e2e/test_body.py [hunk 1]
```diff
@@ -1,15 +1,35 @@
...
 from vital import Client
 
 
-def test_body_returns_data(test_client: Client, user_id: str, start_date, end_date):
-    data = test_client.Body.get(user_id, start_date, end_date)
+@pytest.mark.parametrize(
+    "region",
+    [
+        "us",
+        # "eu"
```
Comment (maitham) [inline tests/api/e2e/test_body.py:10]: is this meant to be commented out?

File: vital/api/link.py [hunk 2]
```diff
@@ -12,7 +12,7 @@ def create(self, user_id: str, provider: Optional[str] = None) -> Mapping[str, s
         :param str user_id: user's id returned by service.
         """
         return self.client.post(
-            "/link/token/", {"user_id": user_id, "provider": provider}
+            "/link/token", {"user_id": user_id, "provider": provider}
```
Comment (maitham) [inline vital/api/link.py:15]: Does this cause any issues for fastapi?

Comment (pazsaragi) [reply vital/api/link.py:15]: I'm not sure I follow? The reason this wasn't an issue was because in app-engine you don't have to be exact with URLs

Comment (pazsaragi) [reply vital/api/link.py:15]: With cloud run you do

**Pull request merged**